symplyphysics/laws/kinematic/vector_projections_from_angle.py

```diff
@@ -0,0 +1,28 @@
+from symplyphysics import (
+    symbols, Eq, pretty, solve, Quantity, units,
+    validate_input, validate_output, expr_to_quantity, sin, cos
+)
+
+# Description
+## Most of cases might be represented in 2-dimensional space with two orthogonal axis - vertical Y and horizontal X. Any vector in this space (velocity, force etc) can be easily
+## transformed to sum of two orthogonal vectors - projections of this vector to axis with help of angle between the vector and the horizontal axis X.
+
+velocity  = symbols('velocity')
+angle = symbols('angle')
+vertical_velocity = symbols('vertical_velocity')
+horizontal_velocity = symbols('horizontal_velocity')
+
+vertical_projection = Eq(vertical_velocity, velocity * sin(angle))
+horizontal_projection = Eq(horizontal_velocity, velocity, cos(angle))
+
+def print():
+    return pretty(vertical_projection, horizontal_projection, use_unicode=False)
+
+
+# is there any way to make this law independent from units? we just need to check if input has the same units as output
```

Comment:
Remove quantities from this function and use general variables. You can check that quantities are same in unit tests.